apps/terminal.py

```diff
@@ -75,6 +75,10 @@ def ssh_copy_id_servers(m):
 
 
 keymap = {
+    "shell Whereami": "pwd ",
+    "shell home": "~/",
+    "shell changedir": "cd",
```

Comment:
this is already implemented on line 91, unless you are intending different semantics

Reply:
yes i see cd is implemented already. i'l remove it from the PR. i can't see pwd or ''~/"  anywhere. can I leave that in?